src/pytorch_lightning/plugins/precision/mixed.py

```diff
@@ -1,26 +0,0 @@
-# Copyright The PyTorch Lightning team.
```

Comment:
TODO: should we deprecate or drop it immediately?

Reply:
There seems to be no direct public usage of it: https://grep.app/search?q=%28MixedPrecision%29